### DISCUSSION TITLE
Uploaded SVG image not showing but linked SVG image showing.

### DISCUSSION THREAD

**Discussion by michaelkowalczyk** [LiaScript/LiaScript#184] [General]
I noticed, 
when uploading an .svg image it does not appear. 
when visiting the direct blob: adress the image appears.
when adding an hosted svg image it appears.

![Screenshot - 13_08_2024 , 11_39_23](https://github.com/user-attachments/assets/447986c2-5fc0-4472-b5b7-a360a168331e)

Is a bug or browser dependend? I use the latest Firefox.

**Comment by andre-dietrich** [comment]
It is a special case, I think ;-) ... It seems that SVG need to be handled differently, when generating Blob-files and URLs. ... Thanks to your issue, this could be resolved quickly ... If you do a hard-reload <kbd>Ctrl</kbd> + <kbd>F5</kbd> of the editor, then the problem shall be gone ...

Thanks for the quick info ...

**Reply by michaelkowalczyk** [reply]
Thanks for fixing Andre.
